PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -471,7 +471,7 @@
 
 #        btagDeepC = Var("bDiscriminator('pfDeepCSVJetTags:probc')",float,doc="CMVA V2 btag discriminator",precision=10),
 #puIdDisc = Var("userFloat('pileupJetId:fullDiscriminant')",float,doc="Pileup ID discriminant",precision=10),
-#        nConstituents = Var("numberOfDaughters()",int,doc="Number of particles in the jet"),
+        nConstituents = Var("numberOfDaughters()",int,doc="Number of particles in the jet"),
```

Comment:
numberOfDaughters() is unsigned int
move int to uint8

Reply:
done